lib/constants.js

```diff
@@ -0,0 +1,112 @@
+/** @constant */
+export const DEFAULT_HEADERS = {
+  // Ensures that the page can't be displayed in a frame,
+  // regardless of where the request came from.
+  // It's a security feature to prevent "clickjacking" attacks,
+  // where an attacker might use an iframe to overlay a legitimate page over a deceptive page.
```

Comment:
```suggestion
  /**
   * Ensures that the page can’t be displayed in a frame,
   * regardless of where the request came from.
   * It’s a security feature to prevent “clickjacking” attacks,
   * where an attacker might use an iframe to overlay a legitimate page over a
   * deceptive page.
   */
```

Actual doc comments, actual unicode punctuation characters instead of typewriter ASCII